### ISSUE TITLE
Build failure with GHC 8

### ISSUE THREAD

**Issue by snoyberg** [albertoruiz/hmatrix#192]
```
> /tmp/stackage-build8$ stack unpack hmatrix-0.17.0.1
Unpacked hmatrix-0.17.0.1 to /tmp/stackage-build8/hmatrix-0.17.0.1/
> /tmp/stackage-build8/hmatrix-0.17.0.1$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup configure --package-db=clear --package-db=global --package-db=/home/stackage/work/builds/nightly/pkgdb --libdir=/home/stackage/work/builds/nightly/lib --bindir=/home/stackage/work/builds/nightly/bin --datadir=/home/stackage/work/builds/nightly/share --libexecdir=/home/stackage/work/builds/nightly/libexec --sysconfdir=/home/stackage/work/builds/nightly/etc --docdir=/home/stackage/work/builds/nightly/doc/hmatrix-0.17.0.1 --htmldir=/home/stackage/work/builds/nightly/doc/hmatrix-0.17.0.1 --haddockdir=/home/stackage/work/builds/nightly/doc/hmatrix-0.17.0.1 --flags=
Configuring hmatrix-0.17.0.1...
> /tmp/stackage-build8/hmatrix-0.17.0.1$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup build
Building hmatrix-0.17.0.1...
Preprocessing library hmatrix-0.17.0.1...
[ 1 of 28] Compiling Internal.Vector  ( src/Internal/Vector.hs, dist/build/Internal/Vector.o )

src/Internal/Vector.hs:205:1: warning: [-Wredundant-constraints]
    • Redundant constraint: RealFloat a
    • In the type signature for:
           asReal :: (RealFloat a, Storable a) =>
                     Vector (Complex a) -> Vector a

src/Internal/Vector.hs:210:1: warning: [-Wredundant-constraints]
    • Redundant constraint: RealFloat a
    • In the type signature for:
           asComplex :: (RealFloat a, Storable a) =>
                        Vector a -> Vector (Complex a)
[ 2 of 28] Compiling Internal.Devel   ( src/Internal/Devel.hs, dist/build/Internal/Devel.o )
[ 3 of 28] Compiling Internal.Vectorized ( src/Internal/Vectorized.hs, dist/build/Internal/Vectorized.o )
[ 4 of 28] Compiling Internal.Matrix  ( src/Internal/Matrix.hs, dist/build/Internal/Matrix.o )

src/Internal/Matrix.hs:278:5: error:
    • Constraint ‘Ord a’ in the type of ‘sortI’
        constrains only the class type variables
      Use ConstrainedClassMethods to allow it
    • When checking the class method:
        sortI :: forall a. (Element a, Ord a) => Vector a -> Vector CInt
      In the class declaration for ‘Element’

src/Internal/Matrix.hs:279:5: error:
    • Constraint ‘Ord a’ in the type of ‘sortV’
        constrains only the class type variables
      Use ConstrainedClassMethods to allow it
    • When checking the class method:
        sortV :: forall a. (Element a, Ord a) => Vector a -> Vector a
      In the class declaration for ‘Element’

src/Internal/Matrix.hs:280:5: error:
    • Constraint ‘Ord a’ in the type of ‘compareV’
        constrains only the class type variables
      Use ConstrainedClassMethods to allow it
    • When checking the class method:
        compareV :: forall a.
                    (Element a, Ord a) =>
                    Vector a -> Vector a -> Vector CInt
      In the class declaration for ‘Element’
```

**Comment by albertoruiz** [comment]
The hotfixes have been released in 0.17.0.2.

**Comment by idontgetoutmuch** [comment]
Works for me. Can this be closed?
